src/pages/index.vue

```diff
@@ -22,10 +22,21 @@
       </h2>
       <VSearchBar
         v-model.trim="searchTerm"
-        class="max-w-[40rem] mt-4 lg:mt-8"
-        :placeholder="featuredSearchText"
+        class="max-w-[40rem] mt-4 lg:mt-8 relative"
+        :placeholder="featuredSearch.term"
```

Comment:
The placeholder shows up for a brief moment at the start of the animation, and it's a different font than the animated text so it has a flash there.

Reply:
We should remove the placeholder altogether from the input. There's no need for it to be there if the `span` is placed over the input. Leaving it in will cause Safari to announce it without context (like tabbing into the input will cause it to say "Universe" without any context if that's what's in the placeholder).